src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectStyles = {};
```

Comment:
Consider using `let` instead of `const` for `objectStyles` since it is being modified within the function.